internal/resource-request-operator/resourceRequest_controller_test.go

```diff
@@ -0,0 +1,78 @@
+package resourceRequestOperator
```

Comment:
May we also test that the controller behaves as expected when a resourceRequest is deleted?